mobile/library/common/data/utility.h

```diff
@@ -23,6 +23,16 @@ Buffer::InstancePtr toInternalData(envoy_data data);
  */
 envoy_data toBridgeData(Buffer::Instance& data, uint32_t max_bytes = 0);
 
+/**
+ * Transform from Buffer::Instance to envoy_data. The `Buffer::Instance` is not drained after
+ * converting it to `envoy_data`.
+ *
+ * @param data, the Buffer::Instance to transform.
+ * @param max_bytes, the maximum bytes to transform or 0 to copy all available data.
+ * @return envoy_data, the bridge transformation of the Buffer::Instance param.
+ */
+envoy_data toBridgeData(const Buffer::Instance& data, uint32_t max_bytes = 0);
```

Comment:
not specific to this PR, but it's a bit confusing that we have `common/bridge/utility.{h,cc}`, then `common/data/utility.{h,cc}` where we also have "bridging" functions.  Maybe we should consolidate at some point, or rename?